### PR TITLE
Rm https section

### PR DESCRIPTION
**Pull request by chriddyp**
@alexandresobolevski - could you review? a couple different things here, check it out commit-by-commit

### CODE REVIEW

File: app/utils/utils.js [hunk 1]
```diff
@@ -46,17 +46,14 @@ export function getQuerystringParam(PARAM) {
...
 export function isOnPrem() {
-    if (isWebBrowser) {
-        // on prem has a specific path for the app; it is simply '/' otherwise
-        return contains('external', baseUrl());
-    }
-    // TODO: is this always the case for the electron process?
-    return false;
+    return true;
+    // on prem has a specific path for the app; it is simply '/' otherwise
+    return (isWebBrowser && contains('external-data-connector', baseUrl()));
```
Comment (alexandresobolevski) [inline app/utils/utils.js]: this return is never reached no?

Comment (chriddyp) [inline app/utils/utils.js]: good catch - fixed

**Comment by chriddyp** [comment]
good to 💃 ?

**Comment by alexandresobolevski** [comment]
heck ye

**Pull request merged**